modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestProcessor.java

```diff
@@ -275,8 +275,10 @@
                     if (secCtx0 == null || ses.isTokenExpired(sesTokTtl))
                         ses.secCtx = secCtx0 = authenticate(req, ses);
 
-                    try(OperationSecurityContext s = ctx.security().withContext(secCtx0)) {
+                    try (OperationSecurityContext s = ctx.security().withContext(secCtx0)) {
                         authorize(req);
+
+                        return handle(req, true);
```

Comment:
I think it's ok to call the handle method here, 
but more readable would move the second call into else block.